src/AWS/Shared/Storage/DynamoDBStorage.cs

```diff
@@ -221,6 +221,10 @@ private async ValueTask CreateTableAsync(string tableName, List<KeySchemaElement
                 TableDescription tableDescription = await TableWaitOnStatusAsync(tableName, TableStatus.CREATING, TableStatus.ACTIVE);
                 tableDescription = await TableUpdateTtlAsync(tableDescription, ttlAttributeName);
             }
+            catch (ResourceInUseException)
```

Comment:
Should the create be wrapped in it's own try/catch if intended to ignore this? As is the create could throw and then the TTL update not applied.

Reply:
Yes, that would be better